service.py

```diff
@@ -15,34 +20,44 @@ class Servicer(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer
              def __init__(self):
                  pass
 
-             #TODO
              def GetStatus(self, request, context):
-                 """Get and return status to the user."""
-                 print("Get Status")
+                 print("[INFO] Requesting GetStatus service")
 
-             #TODO
-             def DownloadZippedFiles(self, request_iterator, context):
-                 """Download zipped files from azrue blob storage."""
-                 if request_iterator.name:
-                     return utility.download_chunk(self.tmp_file_name)
+                 if(grpc.StatusCode.OK.value[0]!= State.AVAILABLE.value):

```

Comment:
You dont have a global variable that holds the state of the service. You are just comparing two constants. Please check this https://github.com/hwsc-org/hwsc-document-svc/blob/master/service/service.go#L25